swatch-core/src/main/java/org/candlepin/subscriptions/db/model/Offering.java

```diff
@@ -146,4 +152,46 @@ public class Offering implements Serializable {
   public Boolean getHasUnlimitedUsage() {
     return hasUnlimitedUsage;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof Offering)) {
+      return false;
+    }
+    Offering offering = (Offering) o;
+    return Objects.equals(sku, offering.sku)
+        && Objects.equals(productName, offering.productName)
+        && Objects.equals(description, offering.description)
+        && Objects.equals(productFamily, offering.productFamily)
+        && Objects.equals(cores, offering.cores)
+        && Objects.equals(sockets, offering.sockets)
+        && Objects.equals(hypervisorCores, offering.hypervisorCores)
+        && Objects.equals(hypervisorSockets, offering.hypervisorSockets)
+        && Objects.equals(role, offering.role)
+        && serviceLevel == offering.serviceLevel
+        && usage == offering.usage
```

Comment:
Is there a reason  not to use Objects.equals here?

Reply:
Since the enums are singletons that == compare should work the same as .equals(). This is also just the default behaviour when you autogenerate the Equals/hashcode from Intellij.